acct-user/cmd5checkpw/metadata.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
This'll be a bit problematic. We wont push new packages without maintainer, but its an acct one....

I think if you want to save this package, youll need to maintain it :\